Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "zed_dot"
+version = "0.1.2"
```

I'll bump the version on a separate commit, let's keep it as-is for now.

```suggestion
version = "0.1.0"
```